### DISCUSSION TITLE
Restarting P8?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1415] [Q&A] [answered]
Has the procedure for restarting the model changed from P7 to P8? Could someone point me to the current documentation?

**Comment by benjamin-cash** [comment]
@DeniseWorthen - Just to follow up on this, since you mentioned this question specifically. As far as I can tell, there is no detailed documentation available that describes all of the changes in going from P7 to P8. Just by eyeballing the various files it is immediately obvious that there are changes, which puts me in the position of knowing that some things have changed but no catalog of what they are. 

The workflow and the regression tests do not work on the community system that I am using for my runs, and running the regression test on Orion or Cheyenne and then porting and updating those files for use on Frontera is a tedious and error-prone operation. It's what I had to do to get the initial tests running but I would really like to avoid doing that every time I need to make a change in the model configuration. 

If that's what it takes I will, but that is a massive barrier to use. And even then I will still need to get questions like the one that started this thread answered, because otherwise I can't be sure I found all of the files that need to be changed, and at the moment this is the only place where I can get questions answered.

**Comment by DeniseWorthen** [comment, marked as answer]
I do not run either the workflow or the prototypes. As far as I know, neither is set up for restarting a run, which is why there is no documentation available on restarting prototypes. 

The only exercise of restarting the global coupled model is w/in the RT system. The basic procedure for restarting the coupled model in the RT has not changed and is documented where it has always been documented: https://github.com/ufs-community/ufs-weather-model/wiki/Restarting-the-Coupled-Model

**Reply by benjamin-cash** [reply]
Thanks @DeniseWorthen! This is pretty much the information I needed to move forward. And sorry for bugging people here with it, if there was another forum where I could get questions answered I would use it. :(

**Reply by benjamin-cash** [reply]
@DeniseWorthen - I did find one apparent change that needs to be made in restarting that is not captured in the documentation. In the regression test nstf_name changes from '2,1,0,0,0' to '2,0,0,0,0' on restarting.

**Reply by DeniseWorthen** [reply]
The ICs for the benchmark P8 regression tests do not contain NSST spin-up so the initial NSTF setting is different in that case than the other tests.

**Comment by benjamin-cash** [comment]
As an FYI, I asked again in EPIC AUS and their response was "In general, users can post on the Discussions page of the repository they need help with."
